server/src/test/java/io/crate/integrationtests/AlterTableIntegrationTest.java

```diff
@@ -269,4 +269,26 @@ private void execute_statements_that_drop_and_add_column_with_the_same_name_agai
             String.format(Locale.ENGLISH, rawRows, newColumnOid, newColumnOid + 1, newColumnOid + 2)
         );
     }
+
+    @Test
```

Comment:
don't we need a new TestCluster here?

Reply:
Ah yes, because `_dropped_2` might get undeterminsitic. 

I updated query to show only last added column to avoid starting u a new cluster.
https://github.com/crate/crate/pull/14813/commits/466174bdaee14b1e389e5e7761fc19f30ffd0b29

We actually want to check that new column get position 3 (actual expected position and not 2 which would have thrown an error).